python/CMakeLists.txt

```diff
@@ -80,6 +80,15 @@ Converters/PycExcp.cc
 Converters/PycImport.cc
 Converters/PycRecord.cc
 Converters/PycValueHolder.cc
+Converters/PycArray.h
```

Comment:
@gijzelaerr, why did you have to add these headers? Do we do the same for Python 3 as well?